src/components/Table/head/header.js

```diff
@@ -98,25 +97,31 @@ export default class Header extends Component {
                 aria-label={this.getHeaderContent()}
             >
                 <StyledWrapper style={headerStyles}>
-                    <StyledHeaderContainer
-                        className="rainbow-table_header-container"
-                        role="presentation"
-                        headerAlignment={headerAlignment}
-                        onClick={this.handleSort}
-                    >
-                        <StyledContent
-                            title={this.getHeaderContent()}
-                            className="rainbow-table_header-content"
+                    <RenderIf isTrue={HeaderComponent}>
+                        {/* eslint-disable-next-line react/jsx-props-no-spreading */}
+                        <HeaderComponent {...rest} onSort={this.handleSort} />
```

Comment:
here we need to rename props, the props need to be the same as main props, for example the column does not have any content props, I assume that this is mapped to header, if so then we need to use header instead
Also I think we should avoid passing sortable since I don't see for what they will need this, it is only used internally to send the onSort event